service/src/main/java/org/apache/griffin/core/job/JobServiceImpl.java

```diff
@@ -661,4 +654,21 @@ private String getSinksPath(String jsonString) {
             return null;
         }
     }
+
```

Comment:
It would be nice to have a test on this method

Reply:
done